openshift/osd-example-operator
# readme update

- Boilerplate: Bootstrap at 84518ed16af54fb1713bc169753c161412c7c78d
- bp update
- operator init
- readme update
- force pipeline
- Update README.md
- bp -update
- harness update
- harness update
- test pipeline
- readme
- bp update
- gitignore
- SDCICD-1209: rename operator
- update owners to current team
- Boilerplate: Update to 4261009734da2324bdbe58f75a7655754a44e41d
- update readme
- update readme
- pipeline check
- bp-update
- misc: update readme to trigger pipeline
